github/users.go

```diff
@@ -166,3 +166,59 @@ func (s *UsersService) ListAll(opt *UserListOptions) ([]User, *Response, error)
 
 	return *users, resp, err
 }
+
+// ListInvitations lists all currently-open repository invitations for the
+// authenticated user.
+//
+// GitHub API docs: https://developer.github.com/v3/repos/invitations/#list-invitations-for-a-repository
```

Comment:
This link should be https://developer.github.com/v3/repos/invitations/#list-a-users-repository-invitations.